packages/api/src/beacon/routes/lightclient.ts

```diff
@@ -88,116 +84,64 @@ export type Endpoints = {
   >;
 };
 
-export function getDefinitions(config: ChainForkConfig): RouteDefinitions<Endpoints> {
-  // this beacon config will be stored here in this closure so fork digests don't need to be recomputed
-  let beaconConfig: BeaconConfig | undefined;
-
+export function getDefinitions(_config: ChainForkConfig): RouteDefinitions<Endpoints> {
   return {
     getLightClientUpdatesByRange: {
       url: "/eth/v1/beacon/light_client/updates",
       method: "GET",
       req: {
-        writeReq: ({startPeriod, count, genesisValidatorsRoot}) => {
-          // store the beacon config if its a new genesisValidatorsRoot
-          if (
-            genesisValidatorsRoot &&
-            (beaconConfig === undefined || !byteArrayEquals(beaconConfig.genesisValidatorsRoot, genesisValidatorsRoot))
-          ) {
-            beaconConfig = createBeaconConfig(config, genesisValidatorsRoot);
-          }
-          return {query: {start_period: startPeriod, count}};
-        },
+        writeReq: ({startPeriod, count}) => ({query: {start_period: startPeriod, count}}),
         parseReq: ({query}) => ({startPeriod: query.start_period, count: query.count}),
         schema: {query: {start_period: Schema.UintRequired, count: Schema.UintRequired}},
       },
-      resp: {
+      resp: JsonOnlyResp({
         data: {
           toJson: (data, meta) => {
-            if (data.length !== meta.versions.length) {
```

Comment:
this check seemed a bit unnecessary this known data we get from server impl
https://github.com/ChainSafe/lodestar/blob/f777b31dcaff6bd44630a802e5fd35e57070a33f/packages/beacon-node/src/api/impl/lightclient/index.ts#L20

I guess it would improve the error message if there is a bug in the server impl but it would latest fail at line 102, I changed the for-loop to go over data instead of meta which seems the better approach.